front-end/src/Profile.css

```diff
@@ -78,3 +77,40 @@
   background-color: #2D2D2D;
   color: #3CDD1B;
 }
+
+#buttons {
+    display: flex;
+    justify-content: space-around;
+}
+
+.filter-button {
+    margin: 2em;
+    font-size: 18px;
+    border: 2px solid #84ED6F;
+    background-color: #666666;
+    color: #84ED6F;
+    padding: 1em 2em 1em 2em;
+}
+
+fieldset{
+  width: 90%;
+  background-color:#2d2d2d;
+  color: #D1CDC7; 
+  margin-left: 25%;
+  margin-right: 25%;
+  text-align: center;
+}
+.showDetails {
+  width: 100%;
+  background-color:#2d2d2d;
+  text-align: center;
+}
+.showContent {
```

Comment:
I would suggest renaming these classes to fit in with the class name style in the rest of the code, so something more like `show-content`